CONTRIBUTING.md

```diff
@@ -510,11 +546,15 @@ def test_my_plotting_case():
 
 ### Documentation
 
+Most documentation sources are in the `doc` folder. The documentation are
```

Comment:
I think this first section should be revised because most of the documentation is in the `examples` folder and the docstrings of the code in the `pygmt` folder.

Reply:
Good point, I've updated it in d5ba4156733d79a89b6631aea3989b2171d468f9 to be clearer hopefully.